bitcoin/Cargo.toml

```diff
@@ -33,7 +33,7 @@ hex = { package = "hex-conservative", version = "0.3.0", default-features = fals
 internals = { package = "bitcoin-internals", path = "../internals", features = ["alloc", "hex"] }
 io = { package = "bitcoin-io", path = "../io", default-features = false, features = ["alloc", "hashes"] }
 primitives = { package = "bitcoin-primitives", path = "../primitives", default-features = false, features = ["alloc", "hex"] }
-secp256k1 = { version = "0.30.0", default-features = false, features = ["hashes", "alloc"] }
+secp256k1 = { version = "0.31.1", default-features = false, features = ["hashes", "alloc", "rand"] }
```

Comment:
In 245e4d6fc2e7d6366b419ceefdec047f7cf2bd68:

Please remove the `"rand"` feature here.

Reply:
Ah woops, done.